packages/discord.js/typings/index.d.ts

```diff
@@ -1503,6 +1503,11 @@ export type Interaction<Cached extends CacheType = CacheType> =
   | AutocompleteInteraction<Cached>
   | ModalSubmitInteraction<Cached>;
 
+export type RepliableInteraction<Cached extends CacheType = CacheType> =
+  | CommandInteraction<CacheType>
+  | MessageComponentInteraction<CacheType>
+  | ModalSubmitInteraction<CacheType>;
+
```

Comment:
`BaseInteraction#isRepliable()` is:

https://github.com/discordjs/discord.js/blob/682e0e1802720ead2e940dbd1723239c691681cb/packages/discord.js/src/structures/BaseInteraction.js#L279-L285

Rather than specifying which types it could be, it excludes ones that are not possible. To be pedantic, maybe reflect the behaviour in the typings too?

```suggestion
export type RepliableInteraction<Cached extends CacheType = CacheType> = Exclude<Interaction, AutocompleteInteraction>;
```

Reply:
Should I add another commit removing it?